resources/instructions/landingPageTemplateChoice.html

```diff
@@ -0,0 +1,262 @@
+                    'landingPageStatus',
+                    {},
+                    handleLandingPageStatusResponse
+                );
+
+                // chooseTemplateButton click
+                const chooseTemplateButtonElement = document.getElementById(
+                    'chooseTemplateButton'
+                );
+                chooseTemplateButtonElement.addEventListener(
+                    'click',
+                    chooseTemplateClicked
+                );
+            });
+
+            function chooseTemplateClicked() {
+                const selectedTemplateElement = document.querySelector(
+                    'input[name="landingPageOption"]:checked'
+                );
+                if (!selectedTemplateElement) {
+                    const globalErrorElement =
+                        document.getElementById('globalError');
+                    globalErrorElement.innerText = 'No landing page selected.';
+                    globalErrorElement.style.display = 'block';
+                } else {
+                    const selectedValue = selectedTemplateElement.value;
+                    webviewMessaging.sendMessageRequest('landingPageChosen', {
+                        landingPageType: selectedValue
+                    });
+                }
+            }
+
+            function handleLandingPageStatusResponse(response) {
+                if (response.error) {
+                    const globalErrorElement =
+                        document.getElementById('globalError');
+                    globalErrorElement.innerText = response.error;
+                    globalErrorElement.style.display = 'block';
+                    const landingPageCollectionElement =
+                        document.getElementById('landingPageCollection');
+                    landingPageCollectionElement.classList.add('disabled');
+                    const radioElements = document.querySelectorAll(
+                        ".enabled-content-toggle > input[type='radio']"
+                    );
+                    radioElements.forEach((radioElement) => {
+                        radioElement.disabled = true;
+                    });
+                    const chooseTemplateButton = document.getElementById(
+                        'chooseTemplateButton'
+                    );
+                    chooseTemplateButton.disabled = true;
+                } else {
+                    for (const landingPageType in response.landingPageCollection) {
+                        const landingPageStatus =
+                            response.landingPageCollection[landingPageType];
+                        if (!landingPageStatus.exists) {
+                            const landingPageLabelId = `${landingPageType}LandingPageTemplateLabel`;
+                            const landingPageLabelElement =
+                                document.getElementById(landingPageLabelId);
+                            if (landingPageStatus.warning) {
```

Comment:
I'm not understanding what this is for as I dont see any warning text defined anywhere?

Reply:
The warning data comes back as one of the status data items from the `TemplateChooserCommand.getLandingPageStatus()` [method](https://github.com/salesforce/salesforcedx-vscode-mobile/pull/48/files#diff-7d148ba5f620ba14e46a900655961c4d76f535280279cdb20e203adbbe52bf45R228-R230). That status check will create a warning if one or both of the landing page files for the given template are not available in the project.